resources/views/_partials/forms/create-course.blade.php

```diff
@@ -0,0 +1,99 @@
+<form action="{{ route('staff.courses.store') }}" method="POST" 
+class="space-y-3" 
+enctype="multipart/form-data"
+x-data="courseForm({{ json_encode(old()) }})">
```

Comment:
```suggestion
x-data="{ type: '{{ old('type') }}' }">
```

Reply:
are we using any reactivity if not we may not need this at all...I guess it is being used for the convenience that `x-model` provides.